lib/src/command/global_activate.dart

```diff
@@ -22,6 +23,11 @@ class GlobalActivateCommand extends PubCommand {
         allowed: ["git", "hosted", "path"],
         defaultsTo: "hosted");
 
+    argParser.addOption("features",
+        abbr: "f",
+        help: "Feature(s) to enable, or disable with - prefix.",
```

Comment:
I think using `-` will be confusing since it's also meaningful for command line options themselves. This looks weird to me:

```
pub global activate foo -f-thing
```

Or, worse:

```
pub global activate foo --features -standalone,-ssh -spath
```

How about `--features` to opt-in and `--omit-features` to opt out?